kicad.spec.template

```diff
@@ -39,12 +41,14 @@ BuildRequires:  libcurl-devel
 BuildRequires:  OCE-devel
 BuildRequires:  openssl-devel
 BuildRequires:  python2-devel
+BuildRequires:  libngspice-devel
 
 # Documentation
 BuildRequires:  asciidoc
 BuildRequires:  po4a
 
 Requires:       electronics-menu
+Requires:       libngspice
```

Comment:
It seems that this is not needed according to https://github.com/KiCad/fedora-packaging/pull/24#issuecomment-451116742